packages/polaris-viz/src/components/LineChartRelational/components/CustomLegend/CustomLegend.scss

```diff
@@ -1,6 +1,8 @@
 .Container {
   display: flex;
+  flex-wrap: nowrap;
   gap: 10px;
-  flex-wrap: wrap;
   list-style: none;
+  margin: 0;
+  padding: 0;
```

Comment:
I noticed some [visual changes](https://shopify.chromatic.com/test?appId=6062ad4a2d14cd0021539c1b&id=675af3ed6f71e74421ed3b5d) against baseline in the chromatic. Do we want the styles to be slightly different?

Reply:
I ended up removing this, I don't think we need them now 👍